packages/forms/src/elements/address/postcodeLookup.tsx

```diff
@@ -42,6 +42,10 @@ export const PostcodeLookup: React.FC<PostcodeLookupProps> = ({
 			onPostcodeChanged(form.getFieldState('postcodeLookup').value);
 	};
 
+	useEffect(() => {
```

Comment:
Do you need to set the ref each time the component renders, or just the first time? If the latter, use an empty array as the second argument to useEffect.